spec/lib/registries/full_topic_taxonomy_registry_spec.rb

```diff
@@ -48,6 +49,12 @@
       fetched_document = registry[first_level_base_path]
       expect(fetched_document['content_id']).to eq(first_level_content_id)
     end
+
+    it "can look up a taxon from the id" do
+      fetched_document = registry.find_parent_by_id(child_content_id)
+      expect(fetched_document[:path]).to eq(child_base_path)
+      expect(fetched_document[:title]).to eq(taxon_title)
+    end
```

Comment:
Would it be possible to add a test to the "when topic taxonomy API is unavailable" context block as well?

Reply:
👍 now added